public/admin/entries/form.haml

```diff
@@ -16,7 +16,6 @@
   .field
     = f.label :body, caption: t("#{@name}.body")
     %br
-    %span{ id: 'upload-disabled-warning', style: 'display: none' }= t('file_upload.notice.unavailable_with_wysiwyg')
```

Comment:
Quill supports file upload. So we can upload files while using wysiwyg if S3 storage is configured.